CVCotM: Clarify the Wii U VC version is unsupported (#4734)
ArchipelagoMW/Archipelago: worlds/cvcotm/docs/setup_en.md

- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases/latest).
- A Castlevania: Circle of the Moon ROM of the US version specifically. The Archipelago community cannot provide this.

> The Castlevania Advance Collection ROM can be used, but it has no audio. The Wii U Virtual Console ROM is unsupported
> and doesn't work at all.

Reviewer: ```suggestion
The Castlevania Advance Collection ROM can be used, but it has no audio. The Wii U Virtual Console ROM does not work.
```
